pf_driver/src/pf/pf_interface.cpp

```diff
@@ -69,6 +70,20 @@ bool PFInterface::handle_version(int major_version, int minor_version)
     return false;
 }
 
+void PFInterface::setup_param_server()
+{
+    if(expected_device_ == "R2000")
```

Comment:
Sorry I am just nitpicking, but the variable name should be `product_`. But both are the same.